### ISSUE TITLE
Confidence Module Output

### ISSUE THREAD

**Issue by ludwigwinkler** [gcorso/DiffDock#267]
Dear authors,

first of all thank you for your work and for releasing everything in an accessible manner for others to incorporate into their research. :)

We're working on top of the confidence predictions.

1) Is my understanding correct that the confidence is modeled by a single logit which we can pass through a sigmoid to obtain a probability of confidence?
It seems to be used as such in the inference code:
https://github.com/gcorso/DiffDock/blob/b4704d94de74d8cb2acbe7ec84ad234c09e78009/inference.py#L278
Upon some searching I found 
https://github.com/gcorso/DiffDock/blob/b4704d94de74d8cb2acbe7ec84ad234c09e78009/confidence/dataset.py#L193
where the labels are scalar quantities per sample and the confidence module predicts a single scalar as well to be trained against here https://github.com/gcorso/DiffDock/blob/b4704d94de74d8cb2acbe7ec84ad234c09e78009/confidence/confidence_train.py#L130

2) We evaluated the confidence logits for the provided examples and got the following values:
Ligand 1a46: roughly -2.11 -> ~10% 
Ligand 6w70: roughly -1.10 -> ~24% 
Ligand 6moa: roughly +0.38 -> ~60%
Do these values seem reasonable?

Thanks again for all the work you've put into this. 

Cheers.
